Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,53 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            string a=equation.Split('*')[0];
+            string b=equation.Split('*')[1].Split('=')[0];
+            string c=equation.Split('=')[1];
+            int result=0;
+            var ans=0.0;
+            String ans1=null, ans2=null;
+            if (a.Contains("?"))

```

Comment:
Refer SOLID principle Single responsibility principle and write meaningful function and variable names. 

Follow this pseudo code for example : 


GetMissingDigit(string multiplier1, string multiplier2, string result)
{
  // Call appropriate method as required.
  
  int multiplier1;
  if(int.TryParse(multiplier1,out multiplier1) == false)
  {
	GetMissingDigitInMultiplier(int, string, int);
  }
  elseif(int.TryParse(multiplier1,out multiplier1) == false)
  {
  
  }
  else
  {
  GetMissingDigitInResult(int,int, string);
  }
}

GetMissingDigitInMultiplier(int multiplier1, string multiplier2, int result)
{

}

GetMissingDigitInResult(int multiplier1, int multiplier2, string result)
{

}

Reply:
Changes are made as Specified